src/Orchard.Web/Modules/Orchard.Projections/FilterEditors/Forms/StringFilterForm.cs

```diff
@@ -36,8 +36,14 @@ public class StringFilterForm : IFormProvider {
                             Title: T("Value"),
                             Classes: new[] { "text medium", "tokenized" },
                             Description: T("Enter the value the string should be.")
-                            )
-                        );
+                        ),
+                        _IgnoreIfEmptyValue: Shape.Checkbox(
+                            Id: "IgnoreFilterIfValueIsEmpty",
+                            Name: "IgnoreFilterIfValueIsEmpty",
+                            Title: "Ignore filter if value is empty",
```

Comment:
Can we add T("Ignore filter if value is empty"), please?

Reply:
Absolutely, good catch!

Reply:
Fixed in 9481a66fbf1ceef908e7161b4e2ced7a76800787.